feature-assets/src/main/java/io/novafoundation/nova/feature_assets/presentation/transaction/history/mixin/OperationMappers.kt

```diff
@@ -171,14 +178,41 @@ private fun substrateCallUi(
     }
 }
 
+private fun Operation.Type.Swap.isIncome(chainAsset: Chain.Asset): Boolean {
+    return chainAsset.fullId == amountOut.chainAsset.fullId
+}
+
+private fun Operation.Type.Swap.formatSubHeader(resourceManager: ResourceManager): CharSequence {
+    val iconColor = resourceManager.getColor(R.color.chip_icon)
+    val chevronSize = resourceManager.measureInPx(12)
+    val arrowRight = resourceManager.getDrawable(R.drawable.ic_arrow_right).apply {
+        setBounds(0, 0, chevronSize, chevronSize)
+        setTint(iconColor)
+    }
+
+    val imageAlignment = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+        ImageSpan.ALIGN_CENTER
+    } else {
+        ImageSpan.ALIGN_BASELINE
+    }
+
+    return buildSpannable(resourceManager) {
+        append(amountIn.chainAsset.symbol)
+        append(" ")
+        appendSpan(ImageSpan(arrowRight, imageAlignment))
```

Comment:
Using `→` symbol works poorly since it is aligned to the bottom and also looks not like on mockups

Reply:
I think it's great solution